Principes/SCProcess.md

```diff
@@ -6,5 +6,53 @@
     A copy of the license is included in the section entitled "GNU
     Free Documentation License".
 
-# Pllug Summer Camp process 
+# PLLUG Summer Camp process 
+
+## Projects 
+
+Optionaly projects should have value for PLLUG or for free software community.
+
+## Glossary 
+
+* **PSC** - PLLUG Summer Camp.
+* **Hard Task Master (mentor)** - Person who responsible for students and project process.
+* **Milestone** - period of project coding should be finished by demo.
```

Comment:
Maybe it will be better to reorganize this sentence for better readability. We also could call that period - "iteration of project development"